modules/workstation-cluster/README.md

```diff
@@ -0,0 +1,158 @@
+# Workstation cluster
+
+This module allows to create a workstation cluster with associated workstation configs and workstations. In addition to this it allows to set up IAM bindings for the workstation configs and the workstations.
+
+## Examples
+
+## Simple example
```

Comment:
Can you add a short description to ach example?

Reply:
Done